src/amo/components/SearchPage.js

```diff
@@ -42,12 +44,16 @@ export default class SearchPage extends React.Component {
       <Paginate LinkComponent={LinkComponent} count={count} currentPage={page}
         pathname={pathname} queryParams={queryParams} showPages={0} />
     ) : [];
+    const searchSort = hasSearchParams && results.length ? (
+      <SearchSort filters={filters} pathname={pathname} />
+    ) : [];
```

Comment:
I agree; the tests are more wordy to write when it's `null` but I'll do it since you mentioned it and I was already irked by it. 😄